src/PrestaShopBundle/Form/Admin/Sell/Product/ProductType.php

```diff
@@ -76,6 +76,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('shipping', ShippingType::class)
             ->add('options', OptionsType::class)
             ->add('seo', SEOType::class)
+            ->add('suppliers', SuppliersType::class, [
```

Comment:
You shouldn't need this, it was not necessary on all other form types Since the template is supposed to call `form_widget(suppliersForm)` the label is not displayed